node/pkg/node/options.go

```diff
@@ -365,7 +365,7 @@ func GuardianOptionWatchers(watcherConfigs []watchers.WatcherConfig, ibcWatcherC
 					return fmt.Errorf("NetworkID already configured: %s", string(wc.GetNetworkID()))
 				}
 
-				watcherName := string(wc.GetNetworkID()) + "watch"
+				watcherName := string(wc.GetNetworkID()) + "_watch"
```

Comment:
Will this mess with the name in the logging? Like `[ethwatch]` -> `[eth_watch]`?

Reply:
Yes it does. I'll be sure to mention the change in the release notes.